contracts/sfc/NodeDriverAuth.sol

```diff
@@ -89,6 +89,11 @@ contract NodeDriverAuth is OwnableUpgradeable, UUPSUpgradeable {
         driver.setBalance(acc, address(acc).balance + diff);
     }
 
+    /// Issue tokens as a counterparty to burnt FTM tokens.
+    function issueTokens(address acc, uint256 diff) external onlySFC {
+        driver.setBalance(acc, address(acc).balance + diff);
+    }
```

Comment:
Why to introduce a new method? It seems removing the `acc == sfc` enforcement from `incBalance()` should be sufficient.
Since there is no similar restriction in issueTokens() there is no reason to have both of these methods.